models/Dependency/Dao.php

```diff
@@ -196,6 +197,64 @@ public function getRequiredBy($offset = null, $limit = null)
         return $requiredBy;
     }
 
+    /**
+     * @param null $orderBy
+     * @param null $orderDirection
+     * @param null $offset
+     * @param null $limit
+     * @return array
+     */
+    public function getRequiredByWithPath($orderBy = null, $orderDirection = null, $offset = null, $limit = null)
```

Comment:
I would still keep offset and limit params in first place to be consistent with `getRequiredBy()`

Reply:
Done